src/rp2_common/pico_lwip/tools/makefsdata.py

```diff
@@ -24,8 +24,8 @@ def process_file(input_dir, file):
         content_type = "application/octet-stream"
 
     # file name
-    data = f"/{file.relative_to(input_dir)}\x00"
-    comment = f"\"/{file.relative_to(input_dir)}\" ({len(data)} chars)"
+    data = f"/{file.relative_to(input_dir).as_posix()}\x00"
+    comment = f"\"/{file.relative_to(input_dir).as_posix()}\" ({len(data)} chars)"
```

Comment:
_Might_ be worth doing a small refactoring here, so that the `file.relative_to(input_dir).as_posix()` is done just once (and assigned to a new variable) ?

Reply:
I agree that this minor refactor would make it a _bit_ cleaner for sure.

Reply:
Perhaps separate refactoring from delimiter handling to make changes more consistent, since there are some format strings that could be refactored ? Perhaps add a new issue for refactoring ?

Reply:
Refactored path posix conversion